anima-jws-voucher.mkd

```diff
@@ -185,20 +195,25 @@ The following figure gives the decoded JWS Protected Header in JSON syntax:
       ]
     }  
 ~~~~
-{: #VoucherGeneralJWSProtectedHeaderFigure title='JWS Protected Header in JSON Syntax' artwork-align="left"}
+{: #VoucherGeneralJWSProtectedHeaderFigure title='Decoded JWS Protected Header Example' artwork-align="left"}
+
+## JWS Signature
+
+The JWS Signature is generated over the JWS Protected Header and the JWS Payload (= UTF-8 encoded JSON Voucher Data) as described in {{Section 5.1 of RFC7515}}.
+The generated JWS Signature is base64url-encoded to become the string value of the "signature" member.
 
 # Privacy Considerations
 
 The Voucher Request reveals the IDevID of the component (Pledge) that is in the process of bootstrapping.
```

Comment:
Is "Voucher Request" and PVR the same thing? Should then also be harmonized terminology-wise or clarified if they are not the same.

Reply:
PVR == Pledge-Voucher-Request and RVR==Registrar-Voucher-Request